great_expectations/checkpoint/checkpoint.py

```diff
@@ -170,7 +170,7 @@ def run(
         # and action_list are considered the "default" validation.
         using_default_validation = not self.validations and not validations
 
-        run_time = run_time or datetime.datetime.now()
+        run_time = run_time or datetime.datetime.now().astimezone()
```

Comment:
```suggestion
        run_time = run_time or datetime.datetime.now(tz=dateutil.tz.gettz())
```

We could also use `dateutil.tz.gettz()` to get the local timezone, and then skip the extra object creation step.
This would require an `import dateutil.tz` for this module.

Reply:
Not a really strong preference here, but my thinking was that we're using dateutil in expectations. I still have hope that eventually the core library dependencies are isolated from expectation dependencies, so I liked keeping to the standard library here.